modules/vault-elb/main.tf

```diff
@@ -38,7 +38,7 @@ resource "aws_elb" "vault" {
     timeout             = "${var.health_check_timeout}"
   }
 
-  tags = "${merge(map("Name", var.name), var.lb_tags)}"
+  tags = "${merge(var.load_balancer_tags, map("Name", var.name))}"
```

Comment:
Should `var.load_balancer_tags` override the default ones?

Reply:
What default ones?

If you mean "Name", no, I don't think so. Since 'Name' is typically used to set the name of the resource I think it would be wrong for var.load_balancer_tags to override Name with a value other than var.name (which is setting the resource name).

That's the same reasoning I followed on https://github.com/hashicorp/terraform-aws-consul/pull/74 with `tags = "${merge(map("Name", var.cluster_name), var.security_group_tags)}"`